npm_and_yarn/helpers/lib/yarn/updater.js

```diff
@@ -104,16 +104,20 @@ async function updateDependencyFiles(directory, dependencies) {
   const readFile = (fileName) =>
     fs.readFileSync(path.join(directory, fileName)).toString();
   let updateRunResults = { "yarn.lock": readFile("yarn.lock") };
-
+  let requiredVersions = [];
   for (let dep of dependencies) {
     for (let reqs of dep.requirements) {
+      if(requiredVersions.indexOf(reqs.requirement) > -1)
+      {
```

Comment:
```suggestion
      if (requiredVersions.indexOf(reqs.requirement) > -1) {
```

We should probably set up a linter that checks this, but this is a but more in line with the style of the rest of the file.